### PR TITLE
Fix styles

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #3222 #3237 

Simulator Screenshot - iPhone 14 Pro - 2023-07-31 at 20 03 34

### CODE REVIEW

**Comment by DmitryBespalov** [comment]
could you add screenshots please?

File: Multisig/UI/Assets/SelectTopUpAddressViewController.swift [hunk 1]
```diff
@@ -88,7 +91,11 @@ class SelectTopUpAddressViewController: LoadableViewController, UITableViewDeleg
         sections = []
 
         sections.append(SectionItems(section: .safeAccount("SAFE ACCOUNT"), items: [Section.SafeAccount.safe]))
-        sections.append(SectionItems(section: .owners("ACCOUNT OWNER KEYS"), items: safe.ownersInfo!.map { Section.Owner.owner($0) }))
+        sections.append(SectionItems(section: .owners("ACCOUNT OWNER KEYS"),
+                                     items: safe.ownersInfo!.map { Section.Owner.owner($0) }))
+        let view = TableHeaderView(frame: CGRect(x: 0, y: 0, width: 0, height: 80))
+        view.set("Choose for which account or can owner key you would like to top up", centered: true)
```
Comment (DmitryBespalov) [inline Multisig/UI/Assets/SelectTopUpAddressViewController.swift]: the sentence isn't complete (or there's a typo here): `or can owner key ... `

File: Multisig/UI/App/ViewControllerFactory.swift [hunk 2]
```diff
@@ -51,6 +51,20 @@ enum ViewControllerFactory {
         return nav
     }
 
+    static func selectTopUpAddress(safe: Safe) -> UIViewController {
+        let vc = SelectTopUpAddressViewController()
+
+        Self.addCloseButton(vc)
+        vc.safe = safe
+        vc.onSelect = { address in
+            vc.dismiss(animated: true, completion: {
```
Comment (DmitryBespalov) [inline Multisig/UI/App/ViewControllerFactory.swift]: `vc` is in the retain cycle

Comment (MouazAlzahabi) [reply Multisig/UI/App/ViewControllerFactory.swift]: Ops, sorry

Comment (DmitryBespalov) [reply Multisig/UI/App/ViewControllerFactory.swift]: what changed? (:

**Review by DmitryBespalov — APPROVED** [review]

**Comment by DmitryBespalov** [comment]
note: this also handles #3237

**Pull request merged**